package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "scripts": {
     "build": "turbo run build",
-    "build:app": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=app",
+    "build:app": "cd packages/sdk && yarn typechain && turbo run build --filter=@stackly/sdk --filter=app --filter=landing",
```

Comment:
Should we have a different build for the landing? We don't need to build the app to have the landing. I suggest `build:landing`

Reply:
fixed